tasks/request_files/request_files.py

```diff
@@ -138,19 +139,32 @@ def task(
         )
         retry_sleep_secs = DEFAULT_RESTORE_RETRY_SLEEP_SECS
 
-    # Get retrieval type
+    # Get retrieval type value from config first
     try:
-        retrieval_type = os.environ[OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY]
-        if retrieval_type not in ("Standard", "Bulk", "Expedited"):
-            msg = (
-                f"Invalid RESTORE_RETRIEVAL_TYPE: '{retrieval_type}'"
-                " defaulting to 'Standard'"
-            )
-            LOGGER.info(msg)
-            retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
+        retrieval_type = event["config"][CONFIG_RESTORE_RETRIEVAL_TYPE_KEY]
```

Comment:
Remember to update function documentation for additional key in config

Reply:
added this in readme.md
CONFIG_RESTORE_RETRIEVAL_TYPE_KEY = 'orcaRestoreTypeOverride'

Reply:
But not in the function documentation for `task`. See on line 84 where the keys of config are listed.
We could also just replace that with a reference the config.json, as it looks like there are other out-of-date keys in that doc string.

Reply:
ow now it makes sense. 

updated to 
'config' (dict): See the function's schemas/config.json for information on config.